src/less/star-rating-select/base/star-rating-select.less

```diff
@@ -0,0 +1,80 @@
+@import "../../mixins/icon/base/icon-mixins.less";
+@import "../../mixins/utility/utility-mixins.less";
+
+div.star-rating-select {
+    cursor: pointer; // this ensures text selection cursor is not used between stars
+    display: inline-block;
+}
+
+span.star-rating-select__radio {
+    display: inline-flex;
+    position: relative;
+    vertical-align: text-top;
+}
+
+span.span.star-rating-select__radio svg {
+    .icon-base-mixin();
+}
+
+span.star-rating-select__radio_icon {
+    display: inline-flex;
+    height: 14px;
+    outline-offset: 1px;
+}
+
+svg.star-rating-select__icon {
+    height: 12px;
+    width: 12px;
+}
+
+input.star-rating-select__radio_control[type="radio"] {
+    cursor: pointer;
+    font-size: 100%;
+    height: 12px;
+    margin: 0;
+    min-width: 12px;
+    opacity: 0;
+    padding: 0;
+    position: absolute;
+    width: 12px;
+    z-index: 1;
```

Comment:
Are these exactly the same as in radio?
Maybe we can make a mixin and use it in both places, like a `radio-input-mixin()`

Reply:
No, it is a mixture of radio styles and star-rating-select specific styles. I decided against a mixin because other styles (copied from radio) are spread out across other declarations. I'm not sold on what value that would really offer to create such granular mixins that are most likely not going to be reused often.